### DISCUSSION TITLE
Periodic Boundary Conditions

### DISCUSSION THREAD

**Discussion by katto01** [thliebig/openEMS-Project#112] [General]
Hello,

Looking for an example for:

1. a unit cell example of an infinite array in rectangular coordinates. say a patch in a lambda/2 by lambda/2 unit cell
2. a unit cell example of a circular array in cylindrical coordinates. say a unitcell for a circular array of grounded dipoles

Thank you

**Comment by thliebig** [comment]
openEMS does not support PBC as this is very difficult and very memory intensive in FDTD (or any time-domain method).

best regards
Thorsten
